app/src/organisms/OnDeviceDisplay/RobotSettingsDashboard/TouchScreenSleep.tsx

```diff
@@ -96,27 +75,13 @@ export function TouchScreenSleep({
         marginTop={SPACING.spacing5}
       >
         {settingsButtons.map(radio => (
-          <React.Fragment key={`sleep_setting_${radio.label}`}>
-            <SettingButton
-              id={radio.label}
-              type="radio"
-              value={radio.value}
-              checked={radio.value === sleepMs}
-              onChange={handleChange}
-            />
-            <SettingButtonLabel
-              htmlFor={radio.label}
-              isSelected={radio.value === sleepMs}
-            >
-              <StyledText
-                fontSize="1.75rem"
-                lineHeight="1.875rem"
-                fontWeight={TYPOGRAPHY.fontWeightRegular}
-              >
-                {radio.label}
-              </StyledText>
-            </SettingButtonLabel>
-          </React.Fragment>
+          <RadioButton
+            key={`sleep_setting_${radio.label}`}
+            buttonLabel={radio.label}
+            buttonValue={radio.value}
+            onChange={handleChange}
+            isSelected={radio.value === sleepMs}
```

Comment:
The current size is 68px but the height in Figma is 84px (5.25rem)
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/Release%3A-Opentrons-Flex-Touchscreen-Designs?node-id=7808-158040&t=YUW2iGF5OAks9tpZ-0


```suggestion
            isSelected={radio.value === sleepMs}
            height="5.25rem"
```

Reply:
Figma says the height should hug, so we do not want a fixed height. In my storybook, the large button is 84px and the small is 68px, same as the designs.

Reply:
Okay, got it.
I think the issue will be solved by adding `radioButtonType='large'` or setting radioBtuttonType = 'large'

![Screenshot 2023-04-21 at 11 28 29 AM](https://user-images.githubusercontent.com/474225/233675750-3b9f0a77-8414-4e3d-9621-9a179c141649.png)